### DISCUSSION TITLE
Stucked on the wrong tailwind configuration even though tried all the combinations.

### DISCUSSION THREAD

**Discussion by AbdulMueed19** [tailwindlabs/tailwindcss#10964] [Help] [answered]
My folder structure:
![image](https://user-images.githubusercontent.com/81902241/230511224-2cd1e90e-3128-45c0-b721-74ce6250b982.png)

My command:
npx tailwindcss -o ./dist/output.css --watch

My Tailwind Configuration:
module.exports = {
    content: [
        './src/**/*.html',
        './src/**/*.js',
        './src/input.css', 
    ],
    // ...
}

**Comment by adamwathan** [comment, marked as answer]
Hey! It looks like you are running your command from inside the `Lab 4 Express` folder, but your `tailwind.config.js` file is in the `Web_Technologies` folder. Try putting your `tailwind.config.js` file in the `Lab 4 Express` folder where you're running the command.

**Reply by AbdulMueed19** [reply]
Tysm! it worked <3
